CreeDictionary/API/schema.py

```diff
@@ -54,6 +62,9 @@ class SerializedSearchResult(TypedDict):
     # Optional: we might not have some preverbs in our database
     preverbs: Tuple[Union[str, SerializedWordform], ...]
 
+    # This omits preverbs and other features displayed elsewhere
+    relevant_tags: Tuple[SerializedLinguisticTag, ...]
+
     # TODO: there are things to be figured out for this :/
     # Sequence of all reduplication tags present, in order
     reduplication_tags: Tuple[str, ...]
```

Comment:
The reduplication tags are `RdplW+` and `RdplS+` - what makes this challenging is how this is presented under the Preverbs label?

We could rewrite `RdplW+` as `Weak reduplication` and `RdplS+` as `Strong reduplication`? Do we give `ongoing, continuing` as the pop-up for `RdplW+`, and `intermittent, repeatedly; again and again; here and there` for `RdplS+`

But what do we link to, similar to the preverbs, or where do we fetch the content for the pop-up? Arok has conveniently/confusingly provided dictionary entries in CW for each reduplicative element, but strictly speaking they're not preverbs, and do not have an origin in independent particles or in Algonquian morphemes. 

Besides that we have `IC+`, which could be represented as 'Initial change' - I don't know what the pop-up would be, perhaps none.

As an initial solution, I'd go for the rewrite above, with the pop-ups.